test/runtests.jl

```diff
@@ -1,14 +1,17 @@
 ENV["MPLBACKEND"]="agg" # no GUI
 
 using PyPlot, PyCall
+using Compat
+
+VERSION >= v"0.7.0" && using Base64
 
 if isdefined(Base, :Test) && !Base.isdeprecated(Base, :Test)
     using Base.Test
 else
     using Test
 end
 
-plot(1:5, 2:6, "ro-")
+plot(collect(1:5), collect(2:6), "ro-")
```

Comment:
This change shouldn't be needed — we are hitting a bug (JuliaPy/PyCall.jl#533) in PyCall that should hopefully be fixed soon.

Reply:
so should i revert this line back, or keep it this way..

Reply:
revert it

Reply:
(Fix merged, will be tagged in https://github.com/JuliaLang/METADATA.jl/pull/17187)